README.md

```diff
@@ -6,9 +6,20 @@
 
 
 # Contentwriting
-Content for the SecretDecks® brand
+Content for the Autonomous Creator Organisation (ACO) - the body - and SecretDecks (SD) - the soul.
```

Comment:
Suggested rewrite

"Content for the **body**  - the Autonomous Creator Organisation (ACO) and for the **soul** - SecretDecks (SD).